src/renderer/react/features/chat/components/ChatMessageList.css

```
@@ -74,6 +74,23 @@
   gap: 10px;
 }
 
+.cy-message__channel-source {
+  display: block;
+  max-width: 100%;
+  overflow: hidden;
+  color: currentColor;
```

Stylelint (17.14.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)